talkative_robot.rb

```diff
@@ -0,0 +1,209 @@
+def get_user_name
+	puts "Hey, what's your name?"
+	gets.chomp
+end
+
+def get_user_age
+	puts "How old are you? "
+	gets.chomp.to_i
+end
+
+def get_user_gender
+	puts "Lemme ask you - are you a M or F? "
+	gets.chomp.chars.first.capitalize
+end
+
+def get_user_author
+	puts "Favorite author?"
+	gets.chomp
+end
+
+name   = get_user_name
+age    = get_user_age
+gender = get_user_gender
+author = get_user_author
```

Comment:
It is confusing to name this `author` and then have another `author` below. This should be named `fav_author`